spec/factories/claims.rb

```diff
@@ -41,6 +41,18 @@
       remitted_to_magistrate { 'no' }
     end
 
+    trait :case_details_with_remittal do
+      ufn { '20150612/001' }
+      main_offence { MainOffence.all.sample.name }
+      main_offence_date { Date.yesterday }
+
+      assigned_counsel { 'no' }
+      unassigned_counsel { 'no' }
+      agent_instructed { 'no' }
+      remitted_to_magistrate { 'yes' }
+      remitted_to_magistrate_date { Date.new(2023, 3, 1) }
+    end
```

Comment:
so traits can be nested... so I would include the case_details trait to avoid duplicate declarations. :) 

or potentially this is just `:with_remittal` and has those fields and you use both trains in the test. :)